systemd/systemd.go

```diff
@@ -259,6 +268,11 @@ func (c *Collector) collect(ch chan<- prometheus.Metric) error {
 	}
 	defer conn.Close()
 
+	err = c.collectBootStageTimestamps(conn, ch)
+	if err != nil {
+		level.Warn(c.logger).Log("msg", "Failed to collect boot stage timestamps", "err", err)
```

Comment:
Hmm, should we make this `Debug` level? This would trigger a message at every scrape if there is a stage error.

Reply:
Adjusted.